src/lgr_models/models/lgr.py

```diff
@@ -232,6 +231,12 @@ def is_set(self):
         return False
 
 
+class TemporaryLgrBase(LgrBaseModel):
```

Comment:
what was the issue with the non-temporary one?

Reply:
From the release notesof Django 3.2:
> Instantiating an abstract model now raises TypeError.

Trying to something like `LgrBaseModel(name=options['reference_lgr'], file=File(ref_lgr_file))` like [in this command](https://github.com/icann/lgr-django/pull/58/commits/15ba3c935c2cc23fd787f3c6207b7f0d8b4e3b9f#diff-53bf3a9b1da46cf61e9bb5b8e7cb2b1b3bae3d5dc9e421f55b24ebac2913d1ed) would raise the exception